src/Umbraco.Core/Persistence/Repositories/ILastSyncedRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Umbraco.Cms.Core.Persistence.Repositories;
+
+public interface ILastSyncedRepository
+{
+    Task<int?> GetInternal();
+
+    Task<int?> GetExternal();
+
+    Task SaveInternal(int id);
+
+    Task SaveExternal(int id);
```

Comment:
Since these are all async methods they should end with async, I.E. `GetInternalAsync` :smile: